content/Teams/yotouch/index.md

```
@@ -0,0 +1,132 @@
+---
+title: YoTouch
+---
+
+# YoTouch
+
+**Country:** Nigeria
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)